src/main/java/org/mqttbee/api/mqtt/mqtt5/message/publish/Mqtt5WillPublishBuilder.java

```diff
@@ -181,7 +181,7 @@ public Mqtt5WillPublishBuilder(@Nullable final Function<? super Mqtt5Publish, P>
     @NotNull
     @Override
     @Deprecated
-    public Mqtt5WillPublishBuilder<P> topicAliasUsage(@NotNull final TopicAliasUsage topicAliasUsage) {
+    public Mqtt5WillPublishBuilder<P> useTopicAlias(@NotNull final TopicAliasUsage useTopicAlias) {
```

Comment:
Minor: the attribute and parameter should stay topicAliasUsage

Reply:
DONE